src/app.rs

```diff
@@ -176,76 +152,86 @@ impl<Ms, Mdl, INodes: IntoNodes<Ms> + 'static, GMs: 'static> App<Ms, Mdl, INodes
-                    AfterMount::new(model).url_handling(UrlHandling::None)
-                }
-            }) as Box<dyn IntoAfterMount<Ms, Mdl, INodes, GMs>>,
+        let app = Self {
+            cfg: Rc::new(cfg::AppCfg::new(
+                util::window().document().expect("get window's document"),
+                // @TODO_B: trim out to remove mount_point field
+                root_element.get_element().expect("get root element"),
+                update,
+                view,
+                base_path,
+            )),
+            data: Rc::new(AppData {
+                model: RefCell::new(None),
+                main_el_vdom: RefCell::new(None),
+                popstate_closure: RefCell::new(None),
+                hashchange_closure: RefCell::new(None),
+                window_event_handler_manager: RefCell::new(EventHandlerManager::new()),
+                // @TODO_B: We are trying to get rid of using SubManager
+                sub_manager: RefCell::new(crate::app::sub_manager::SubManager::new()),
+                msg_listeners: RefCell::new(Vec::new()),
+                scheduled_render_handle: RefCell::new(None),
+                after_next_render_callbacks: RefCell::new(Vec::new()),
+                render_info: Cell::new(None),
+            }),
         };
-        let app = Self::new(
-            update,
-            None,
-            view,
-            root_element,
-            None,
-            None,
-            Some(app_init_cfg),
-            base_path,
-        );
-        app.run()
-    }
 
-    /// Creates a new `AppBuilder` instance. It's the standard way to create a Seed app.
-    ///
-    /// Then you can call optional builder methods like `routes` or `sink`.
-    /// And you have to call method `build_and_start` to build and run a new `App` instance.
-    ///
-    /// _NOTE:_ If your `Model` doesn't implement `Default`, you have to call builder method `after_mount`.
-    ///
-    /// # Example
-    ///
-    /// ```rust,no_run
-    ///fn update(msg: Msg, model: &mut Model, _orders: &mut impl Orders<Msg, GMsg>) {
-    ///   match msg {
-    ///       Msg::Clicked => model.clicks += 1,
-    ///   }
-    ///}
-    ///
-    ///fn view(model: &Model) -> impl IntoNodes<Msg> {
-    ///   vec![
-    ///       button![
-    ///           format!("Clicked: {}", model.clicks),
-    ///           simple_ev(Ev::Click, Msg::Clicked),
-    ///       ],
-    ///   ]
-    ///}
-    ///
-    ///App::builder(update, view)
-    /// ```
-    pub fn builder(
-        update: UpdateFn<Ms, Mdl, INodes, GMs>,
-        view: ViewFn<Mdl, INodes>,
-    ) -> AppBuilder<Ms, Mdl, INodes, GMs, UndefinedInitAPI> {
-        // @TODO: Remove as soon as Webkit is fixed and older browsers are no longer in use.
-        // https://github.com/David-OConnor/seed/issues/241
-        // https://bugs.webkit.org/show_bug.cgi?id=202881
-        let _ = util::document().query_selector("html");
+        app.data
```

Comment:
Seeking advice: Unsure of what removing this would do.

Reply:
It will probably leave currently existing dom nodes in the mount point alone dom node and append to the contents of the node like so:

Sent from server:
```
<div class="mount"><div id="hahaha" /></div>
```
and after seed starts rendering:
```
<div class="mount"><div id="hahaha" /><div id="app-thing-0" /><div id="app-thing-1" /></div>
```

Originally, `MountType::Takeover` removes the #hahaha div while the other mount type left the #hahaha div alone. The changes here seems to be forcing to the former behavior if I'm reading the modifications to `bootstrap_vdom` correctly. This chain is sort of the whole point of the changes made in #227, after all.

Unless `rerender_vdom` does things differently now? It's been a while since I was here, after all.